packages/flutter/lib/src/painting/image_resolution.dart

```diff
@@ -117,6 +117,10 @@ const double _kLowDprLimit = 2.0;
 /// AssetImage('icons/heart.png')
 /// ```
 ///
+/// For an example on how an [AssetImage] would be resolved to obtain a
+/// [dart:ui.Image] object, refer to the sample code in the documentation for
+/// [ImageProvider].
```

Comment:
I am not sure I understand how the sample code in ImageProvider explains how to obtain a ui.Image from an AssetImage. Right now this doc leaves me with more questions than answers...

Reply:
@goderbauer I thought that since AssetImage implements ImageProvider it must be responsible for getting that ui.Image from an  AssetIamge 😅

Reply:
The code there doesn't use an AssetImage, so it doesn't really show what the bug is asking for. Sure, some of the code in that example can be used to achieve that goal. But you really have to know what you're looking for. It would probably be best to have some sample code here that only shows the relevant bits.

> What #78088 is asking for is for sample code showing how to actually get a ui.Image object out of an AssetImage.